website/static/files/build_orca_reconcile.sql

```diff
@@ -60,9 +60,8 @@ BEGIN
   , last_update          	timestamp with time zone NOT NULL
   , size_in_bytes        	int8 NOT NULL
   , storage_class        	text NOT NULL
-  , delete_marker        	bool NOT NULL
```

Comment:
The AWS docs I looked at were inaccurate, and implied this would be in the report. May need revisiting once we start looking at deletes.

Reply:
So per the docs this is only around when listing a versioned bucket. See `Delete Marker` explanation here https://docs.aws.amazon.com/AmazonS3/latest/userguide/storage-inventory.html. See my comments on adding/mapping a column to the initial temp table to filter on.

Reply:
... That isn't even the right name. These docs are awful. Alright, I'll add it back in.